docs/source/installation.rst

```diff
@@ -374,19 +390,29 @@ And build sktime from this environment:
 
     python setup.py install
 
-Replace ``x64`` by ``x86`` to build for 32-bit Python.
-
 Some users have experienced issues when installing NumPy, particularly version 1.19.4. Note that a recent Windows update may affect compilation using Visual Studio (see `Windows update issue <https://developercommunity.visualstudio.com/content/problem/1207405/fmod-after-an-update-to-windows-2004-is-causing-a.html>`_).
 
+Setting up a Conda environment
+""""""""""""""""""""""""""""""
 If you run into a problem installing the development version and are using Anaconda, try:
```

Comment:
is this section really only "if you run into a problem"? It looks like this condition should be removed, since it appears like it's the standard workflow, and not conditional "ran into problem" coming from another section.

Reply:
Yes, I wasn't sure how to handle this, so I left the old instruction up which just runs `python setup.py install` to install sktime. I did this as I wanted to avoid having introducing a dependency on anaconda for the windows installation. I have now changed this so these instructions are the default but added some preamble saying these instructions are for anaconda but the process will be similiar for any virtual environment manager.

Reply:
makes sense.